### ISSUE TITLE
Extend the Autocomplete feature with extra options

### ISSUE THREAD

**Issue by mistermark** [sourcejs/Source#172]
At the moment I'm overwriting the Autocomplete feature manually with some options that could be extended on top of the existing ones:

- position: absolute/fixed/relative
- top: enter a value or clear the value to allow positioning using CSS
- left: enter a value or clear the value to allow positioning using CSS

Also allow these options to be overwritten on /user level using a config.

**Comment by operatino** [comment]
Agree. Position should be configurable, when top/left coordinates should be calculated depending on input position.

This was originally Ilya's @cheshirsky (LP team) module, asking him directly could speed up the progress. Anyway, I will plan this in some of the nearest updates.
